CHANGES/5165.feature

```diff
@@ -0,0 +1 @@
+Sync all collections (a full mirror) from Galaxy.
```

Comment:
Can you also add a `5165.removal` talking about the removal of the whitelist field.